libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator+Internal.h

```diff
@@ -23,32 +23,37 @@
  */
 
 #import "SFOAuthCoordinator.h"
-
+#import "SFSDKAuthSession.h"
 @class SFOAuthInfo;
 
 typedef NS_ENUM(NSUInteger, SFOAuthTokenEndpointFlow) {
     SFOAuthTokenEndpointFlowNone = 0,
     SFOAuthTokenEndpointFlowRefresh,
     SFOAuthTokenEndpointFlowAdvancedBrowser
 };
+NS_ASSUME_NONNULL_BEGIN
 
 @interface SFOAuthCoordinator ()
 
 @property (assign) BOOL authenticating;
-@property (nonatomic, strong, readonly) NSURLSession *session;
-@property (nonatomic, strong) NSMutableData *responseData;
+@property (nonatomic, strong, readonly, nullable) NSURLSession *session;
```

Comment:
Should these be `nullable`? I feel like at least some of these should be `nonnull`, like `NSURLSession` and `WKWebView`, for instance. One can't authenticate or perform any actions without either of those.

Reply:
Agree, these were always nullable. In introduced the NS_ASSUME_NONNULL to suppress warnings. These are internal properties. Session is also nullified/cleared at some point.